radio/src/targets/taranis/lcd_driver_spi.cpp

```diff
@@ -204,6 +208,10 @@ void lcdRefresh(bool wait)
 
 #if LCD_W == 128
   uint8_t * p = displayBuf;
+  #if defined(RADIO_LR3PRO)
+    // add offset 2px because the driver (SH1106) of the 1.3 OLED is for a 132 display
+    lcdWriteCommand(0x02);
```

Comment:
We need it more generically as. Define offsets and apply them.

Reply:
Again, a generic define please... i.e. a suitably named define... (`LCD_W_OFFSET` maybe?) can be tested for and applied if present... and you can then define it in the `CMakeLists.txt` with all your other radio parameters.